schema/fmi3InterfaceType.xsd

```diff
@@ -72,11 +72,7 @@ ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     </xs:complexType>
     <xs:complexType name="fmi3ScheduledExecution">
         <xs:complexContent>
-            <xs:extension base="fmi3InterfaceType">
-                <xs:attribute name="maxOutputDerivativeOrder" type="xs:unsignedInt" default="0"/>
-                <xs:attribute name="providesIntermediateUpdate" type="xs:boolean" default="false"/>
-                <xs:attribute name="recommendedIntermediateInputSmoothness" type="xs:int" default="0"/>
-            </xs:extension>
+            <xs:extension base="fmi3InterfaceType"/>
```

Comment:
That's not really PR related is it?! ;)

Reply:
No, I think this is: we need to drop those attributes from SE.